cobalt/browser/browser_module.cc

```diff
@@ -1809,7 +1809,10 @@ void BrowserModule::InstantiateRendererModule() {
 void BrowserModule::DestroyRendererModule() {
   DCHECK(task_runner_->RunsTasksInCurrentSequence());
   DCHECK(renderer_module_);
-
+  base::task_runner_util::PostBlockingTask(
+      web_module_->task_runner(), FROM_HERE,
+      base::Bind(&WebModule::WaitForItall,
+                 base::Unretained(web_module_.get())));
   screen_shot_writer_.reset();
```

Comment:
Tiny nit, this screenshot writer could probably be shut down before the postblockingtask. 

Slight tangent, we have pending bug not to have this thing : b/284676534